Cubical/HITs/Hopf.agda

```diff
@@ -5,29 +5,383 @@ open import Cubical.Core.Primitives
 open import Cubical.Core.Prelude
 open import Cubical.Core.Glue
 
+open import Cubical.Foundations.Equiv
+
+open import Cubical.Data.Int
+open import Cubical.Data.Prod
+
 open import Cubical.HITs.S1
 open import Cubical.HITs.S2
+open import Cubical.HITs.S3
 open import Cubical.HITs.Susp
+open import Cubical.HITs.Join
+
+data Interval : Set where
```

Comment:
This is already in https://github.com/agda/cubical/blob/master/Cubical/HITs/Interval/Base.agda